.mocharc.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+	"reporter": "dot",
+	"recursive": true,
+	"timeout": 5000
+}
```

Comment:
this file needs a trailing  ., and i prefer rc files not to have extensions.

Reply:
Ok but it needs an extension in mocha following the [documentation](https://mochajs.org/#priorities):
```
.mocharc.js
.mocharc.yaml
.mocharc.yml
.mocharc.jsonc
.mocharc.json
```
Pick the one you prefer. :)

Reply:
since we can't upgrade mocha past mocha 3, and mocha 3 supports `test/mocha.opts`, which we already [have](https://github.com/benmosher/eslint-plugin-import/blob/master/test/mocha.opts), i'm not sure we have to pick?